src/GitTfs.Vs2022/paket.references

```diff
@@ -0,0 +1,7 @@
+structuremap
+
+group VS2022
+Microsoft.TeamFoundationServer.Client
+Microsoft.TeamFoundationServer.ExtendedClient
+Microsoft.VisualStudio.Setup.Configuration.Interop
+Microsoft.VisualStudio.Settings.16.0
```

Comment:
There is no Microsoft.VisualStudio.Settings.16.0 nuget package avalable on nuget.org

Reply:
Furthermore, the new group VS2022 (not visible in the diff here) needs to be added to the `paket.dependencies` file, too. For reproducability, we also want the paket.lock file to be updated with the concrete versions.

Reply:
Fixed